trieste/acquisition/multiobjective/__init__.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 The Trieste Contributors
```

Comment:
Can we have the acquisition function in the same module as the others? I do agree it's worth splitting the acquisition functions up, but I want to do that in one go so we can choose which ones go together